src/sharedPrompt.ts

```diff
@@ -1,4 +1,4 @@
-export const SHARED_PROMPT = `You are an expert programmer, and you are trying to summarize a git diff.
+export const SHARED_PROMPT = `You are an expert programmer tasked with analyzing a Git diff and providing advice and improvements to optimize its performance. Summarize your findings and provide a recommendation for how to best improve the Git diff.
```

Comment:
GPT summary of [b22112](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/50a2e65d8816904f457cb4d5390f2804a4cfd5a7/src/sharedPrompt.ts#b22112885d9939a64218542e9236b05a8ba1ba93) - [9c4b43](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/b0d24914146721c2c98a413653af9d3da434f9f1/src/sharedPrompt.ts#9c4b43dc799a0e5ed3f05b64b3f1e55ebfe1b543):
* Changed the prompt text in the `SHARED_PROMPT` constant from "You are an expert programmer, and you are trying to summarize a git diff." to "You are an expert programmer tasked with analyzing a Git diff and providing advice and improvements to optimize its performance. Summarize your findings and provide a recommendation for how to best improve the Git diff."